### PR TITLE
Disable recording if perf is not installed

### PR DESCRIPTION
**Pull request by nwrogers**
Fixes: #131

### CODE REVIEW

File: src/recordpage.cpp [hunk 1]
```diff
@@ -88,8 +89,26 @@ RecordType selectedRecordType(const QScopedPointer<Ui::RecordPage> &ui)
     return ui->recordTypeComboBox->currentData().value<RecordType>();
 }
 
+bool isPerfInstalled()
+{
+    QByteArray envPath = qgetenv("PATH");
```
Comment (krf) [inline src/recordpage.cpp]: Should probably just use http://doc.qt.io/qt-5/qstandardpaths.html#findExecutable

=> `const bool isPerfInstalled = !QStandardPaths::findExecutable("perf").isEmpty()`

**Comment by krf** [comment]
There should probably be a central function for getting the actual path to the perf executable? Dunno Hotspot code enough, though.

**Comment by milianw** [comment]
+1, but please resolve the conflicts (I've added a clang-format file now)

**Comment by milianw** [comment]
merged now, thanks @nwrogers

**Pull request closed without merging**